frontend/src/components/NavBar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./NavBar.scss";
+
+function Nav_bar() {
+  return (
+    <header>
+      <div className="navBar">
```

Comment:
Hmm. That's just a suggestion, but I think you could have given the `navBar` class to your header, or used a `<nav>` tag to mark your navigation menu